website/screens/theme-generator/themes/advanced-theme.json

```diff
@@ -441,10 +441,6 @@
     "overlayColor": "#000000b3",
     "overlayOpacity": "0.7",
     "overlayZindex": "1600",
-    "paddingTop": "0px",
```

Comment:
Why did you remove these tokens? They are present in both the component and the variables file:

![image](https://github.com/dxc-technology/halstack-react/assets/44321109/41220ec3-5acf-42b6-b4d2-9462af3db7ee)

Reply:
I forgot to add them again! Thanks!!